internal/app/app.go

```diff
@@ -37,7 +37,7 @@ type githubClient interface {
 	GetLatestCommitFromBranch(ctx context.Context, owner, repo, branch string) (string, error)
```

Comment:
Ensure to document the new parameters `startSide` and `side` in the interface method to clarify their usage and requirements.